src/System.IO.IsolatedStorage/tests/Resources/Strings.resx

```diff
@@ -0,0 +1,261 @@
+  </data>
+  <data name="IO_FileExists_Name" xml:space="preserve">
+    <value>The file '{0}' already exists.</value>
+  </data>
+  <data name="UnauthorizedAccess_IODenied_Path" xml:space="preserve">
+    <value>Access to the path '{0}' is denied.</value>
+  </data>
+  <data name="UnauthorizedAccess_IODenied_NoPathName" xml:space="preserve">
+    <value>Access to the path is denied.</value>
+  </data>
+  <data name="DriveNotFound_Drive" xml:space="preserve">
+    <value>Could not find the drive '{0}'. The drive might not be ready or might not be mapped.</value>
+  </data>
+  <data name="UnknownError_Num" xml:space="preserve">
+    <value>Unknown error '{0}'.</value>
+  </data>
+  <data name="IsolatedStorage_ApplicationMissingIdentity" xml:space="preserve">
+    <value>Unable to determine application identity of the caller.</value>
+  </data>
+  <data name="IsolatedStorage_ApplicationNoEvidence" xml:space="preserve">
+    <value>Unable to determine identity of application.</value>
+  </data>
+  <data name="IsolatedStorage_ApplicationUndefined" xml:space="preserve">
+    <value>Application cannot be determined on an Assembly or Domain store.</value>
+  </data>
+  <data name="IsolatedStorage_AssemblyMissingIdentity" xml:space="preserve">
+    <value>Unable to determine assembly of the caller.</value>
+  </data>
+  <data name="IsolatedStorage_AssemblyNoEvidence" xml:space="preserve">
+    <value>Unable to determine identity of assembly.</value>
+  </data>
+  <data name="IsolatedStorage_AssemblyUndefined" xml:space="preserve">
+    <value>Assembly cannot be determined for an Application store.</value>
+  </data>
+  <data name="IsolatedStorage_CurrentSizeUndefined" xml:space="preserve">
+    <value>Current size cannot be determined for this store.</value>
+  </data>
+  <data name="IsolatedStorage_DeleteDirectories" xml:space="preserve">
+    <value>Unable to delete; directory or files in the directory could be in use.</value>
+  </data>
+  <data name="IsolatedStorage_DomainMissingIdentity" xml:space="preserve">
+    <value>Unable to determine domain of the caller.</value>
+  </data>
+  <data name="IsolatedStorage_DomainNoEvidence" xml:space="preserve">
+    <value>Unable to determine the identity of domain.</value>
+  </data>
+  <data name="IsolatedStorage_DomainUndefined" xml:space="preserve">
+    <value>Domain cannot be determined on an Assembly or Application store.</value>
+  </data>
+  <data name="IsolatedStorage_OnlyIncreaseUserApplicationStore" xml:space="preserve">
+    <value>Increasing the quota of this scope is not supported.  Only the user application scope’s quota can be increased.</value>
+  </data>
+  <data name="IsolatedStorage_Scope_Invalid" xml:space="preserve">
+    <value>Invalid scope.</value>
+  </data>
+  <data name="IsolatedStorage_Scope_U_R_M" xml:space="preserve">
+    <value>Invalid scope, expected User, User|Roaming or Machine.</value>
+  </data>
+</root>
```

Comment:
I could filter them out to the ones that matter, but I figured it wasn't worth it for a test assembly. The way we generate resources I couldn't link to the other either.